pkg/features/feature.go

```diff
@@ -132,6 +132,12 @@ var (
 		true,
 		true,
 		true)
+	PreBootstrap = newFeature(
```

Comment:
this should be called `ProvisioningPreBootstrap` I think

Reply:
also need to indicate that it is only applicable to the cluster-agent and should not be set on a rancher management cluster (local)